src/app/admin/modules/data-management/components/visual-graph-param-selector/visual-graph-param-selector.component.ts

```diff
@@ -0,0 +1,106 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, Validators, ValidatorFn, FormGroup, ValidationErrors} from '@angular/forms';
+import { MatOptionType } from 'src/app/shared/interfaces/mat-option-type';
+import { ParamSelector } from 'src/app/shared/interfaces/visual-graph-param-selector';
+import { StatisticsType } from 'src/app/shared/enums/statistics-type.enum';
+import { StaffGroupingType } from 'src/app/shared/enums/staff-grouping-type.enum';
+import { TraineeNumberCoverageGroupingType } from 'src/app/shared/enums/trainee-number-coverage-grouping-type.enum';
+import { TrainingHoursWorkloadGroupingType } from 'src/app/shared/enums/training-hours-workload-grouping-type.enum';
+
+export const timeValidator: ValidatorFn = (control: FormGroup): ValidationErrors | null => {
+    const selectedStartYear = control.get('selectedStartYear');
+    const selectedEndYear = control.get('selectedEndYear');
+
+    return selectedStartYear.value && selectedEndYear.value && selectedStartYear.value > selectedEndYear.value ?
+           { timeValidator: true } : null;
+};
+
+
+@Component({
+  selector: 'app-visual-graph-param-selector',
+  templateUrl: './visual-graph-param-selector.component.html',
+  styleUrls: ['./visual-graph-param-selector.component.css']
+})
+
+export class VisualGraphParamSelectorComponent implements OnInit {
+
+  selectedGraphValues: ParamSelector;
```

Comment:
`ParamSelector` sounds like a component name to me, consider using `DataGraphConfiguration`